compiler/luci-interpreter/src/core/RuntimeGraph.cpp

```diff
@@ -73,6 +155,10 @@ void RuntimeGraph::execute() const
     // TODO The `configure` method should only be called if the outputs of an operator need to be
     //  resized.
     kernel->configure();
+#if 0
```

Comment:
Need to mark it with TODO, so we can find this place later
```suggestion
// TODO decide where to allocate memory, and uncomment/remove this if
#if 0
```